src/data_processing/instrument_process.py

```diff
@@ -101,8 +102,15 @@ def process_ct25k(self):
         shutil.move(full_path, self.base.daily_file.name)
         self._call_ceilo2nc('ct25k')
 
+    def process_halo_doppler_lidar_calibrated(self):
+        self._process_halo_lidar('-calibrated')
+
     def process_halo_doppler_lidar(self):
-        full_path, self.uuid.raw = self.base.download_instrument('halo-doppler-lidar', largest_only=True)
+        """This can be removed at some point."""
+        self._process_halo_lidar()
+
+    def _process_halo_lidar(self, suffix: Optional[str] = None):
+        full_path, self.uuid.raw = self.base.download_instrument(f'halo-doppler-lidar{suffix}', largest_only=True)
```

Comment:
Wouldn't this try to download `halo-doppler-lidarNone` when `suffix` is not provided?

Reply:
Whoops, good point!